pkg/services/tree/replicator.go

```diff
@@ -130,6 +130,71 @@ func (s *Service) pushToQueue(cid cidSDK.ID, treeID string, op *pilorama.LogMove
 	}
 }
 
+func (s *Service) synchronize(ctx context.Context) {
+	if !s.nodeState.IsOnline() {
```

Comment:
May be it is better to use `tree.SynchronizeAll` in some callback executed when an epoch has ticked?